packages/launchpad/cypress/e2e/support/e2eSupport.ts

```diff
@@ -2,3 +2,4 @@
 require('../../../../frontend-shared/cypress/e2e/support/e2eSupport')
 
 require('./dropFileWithPath')
+require('cypress-plugin-tab')
```

Comment:
:eyes: did some reading about this, interesting. We've also [recommended](https://github.com/cypress-io/cypress/issues/311#issuecomment-787508664) https://github.com/dmtrKovalenko/cypress-real-events previously, but there's no firefox support there

Reply:
Yeah...We have is issue logged https://github.com/cypress-io/cypress/issues/299 in the backlog for tab support so hopefully we can implement & provide first-class support soon.